web_app/contract_tools/airdrop.py

```diff
@@ -49,9 +54,9 @@ def _validate_response(data: List[dict]) -> AirdropResponseModel:
         for item in data:
             validated_item = AirdropItem(
                 amount=item["amount"],
-                proof=item["proof"],
+                proof=item["proof"],  # This is correct now as AirdropItem expects List[str]
                 is_claimed=item["is_claimed"],
                 recipient=item["recipient"]
             )
             validated_items.append(validated_item)
-        return AirdropResponseModel(airdrops=validated_items)
+        return AirdropResponseModel(airdrops=validated_items)
```

Comment:
add blank line in the end of the file
<img width="601" alt="Screenshot 2024-10-30 at 7 58 10" src="https://github.com/user-attachments/assets/e8866b12-a661-42d8-b2ba-49992e302c9d">